src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Build/LanguageServiceErrorListProvider.cs

```diff
@@ -54,7 +54,7 @@ private async Task<AddMessageResult> AddMessageCoreAsync(TargetGeneratedError er
             if (!TryExtractErrorListDetails(error.BuildEventArgs, out ErrorListDetails details) || string.IsNullOrEmpty(details.Code))
                 return AddMessageResult.NotHandled;
 
-            bool handled = await _projectContextHost.OpenContextForWriteAsync(accessor =>
+            bool? handled = await _projectContextHost.OpenContextForWriteAsync(accessor =>
```

Comment:
@ocallesp was there a warning here before? The method appears to return `Task<T?>`, so I would have expected the old code to fail the build.